backend/config/curation-api.yml

```diff
@@ -30,7 +37,7 @@ paths:
             type: string
             example: "<YOUR-CURATOR-API-KEY>"
           required: true
-          description: Your curation API key.
+          description: Your curation API key. A curator key can be generated using the CELLxGENE Discover UI.
```

Comment:
I'm on the fence because this UI is not accessible to non-curators and may raise more questions than it answers. @jahilton - do you have suggestions for how we inform curators (out of band demo on the curator call for example) about the process for generating their API key?

Reply:
Yes, we can do an initial demo on a curator call & follow-up individually with anyone that misses it, has questions, etc.

Reply:
@Bento007 - would you strike this sentence? 

~A curator key can be generated using the CELLxGENE Discover UI.~

Reply:
```suggestion
          description: Your curation API key.```